servers/visual_server.cpp

```diff
@@ -333,7 +333,7 @@ Error VisualServer::_surface_set_data(Array p_arrays, uint32_t p_format, uint32_
 
 	uint8_t *vw = r_vertex_array.ptrw();
 
-	uint8_t *iw;
+	uint8_t *iw = NULL;
```

Comment:
Won't it be a problem at `copymem(&iw[i * 2], &v, 2);` on line 636?

Reply:
Which is a null pointer dereference, you should not do that :P

Reply:
Right, so I'll keep my extra null check added in https://github.com/godotengine/godot/pull/37351/files#r399207194